examples/ex_firecracker/oas_client_gen.go

```diff
@@ -969,13 +969,13 @@ func (c *Client) PatchMachineConfiguration(ctx context.Context, request OptMachi
 				if err := request.Value.Validate(); err != nil {
 					return err
 				}
-				return nil
+				return nil // return 1
 			}(); err != nil {
 				return err
 			}
 		}
-		return nil
-		return nil
+		return nil // return 2
```

Comment:
here you can see, that in this and the next line two `return nil` are generated.